app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelChairAccessPublicTransport.java

```diff
@@ -19,7 +19,8 @@
 	@Override protected String getTagFilters()
 	{
 		return " nodes, ways, relations with (public_transport=platform or " +
-				"(highway=bus_stop and public_transport!=stop_position))" +
+				"(highway=bus_stop and public_transport!=stop_position) or " +
+				" railway ~ station|subway_entrance)" +
```

Comment:
changing "nodes, ways, relations" to "elements" led to a crash complaining that nodes, ways or relations was expected. I've removed the extraneous brackets, though.